tmt/steps/prepare/artifact/providers/info.py

```diff
@@ -43,7 +44,9 @@ class RpmArtifactInfo(ArtifactInfo):
     Represents a single RPM package.
     """
 
-    PKG_URL = "https://kojipkgs.fedoraproject.org/packages/"  # For actual package downloads
+    BASE_URL = os.getenv("RPM_BASE_URL", "https://kojipkgs.fedoraproject.org/packages").rstrip(
+        "/"
+    )  # For actual package downloads
```

Comment:
Btw, these are instance variables, not class-only (because of `@container`, a.k.a. `@dataclass`). Is that intended, I don't know. If you want to make it a class-var, annotate it with `typing.ClassVar[str]`

Reply:
They are instance variables, they shouldn't be, but they need to change anyway to support fmf/CLI, so they will most likely go away anyway. Letting them in for now.